docs/production/upgrade-or-modify.md

```diff
@@ -102,6 +102,43 @@ git_repo_url = https://github.com/zulip/zulip.git
 See also our documentation on [upgrading
 docker-zulip](https://github.com/zulip/docker-zulip#upgrading-from-a-git-repository).
 
+## Updating configuration files
+
+Zulip installations often upgrade many times over their lifetime, and
+we strive to keep all configuration files backwards-compatible.
+However, our practice of leaving the `/etc/zulip/settings.py`
+unchanged during upgrades means that there may be new features which
+are not documented in that file, since it was based on a template
+provided by an earlier version of Zulip, during the initial install.
+
+After upgrading across major versions of Zulip Server, we recommend
+comparing your `/etc/zulip/settings.py` file to the current settings
+template, which can be found in
+`/home/zulip/deployments/current/zproject/prod_settings_template.py`. We
+suggest using that updated template to update
+`/etc/zulip/settings.py`:
+
+1. Ensure you have a
+   [backup](../production/export-and-import.html#backups) of your
+   `/etc/zulip/settings.py` file.
+
+1. Copy
+   `/home/zulip/deployments/current/zproject/prod_settings_template.py`
+   to `/etc/zulip/settings-new.py`.
```

Comment:
Should we just give a `cp` command, maybe collapsing this with the previous step?